advisor-ui/src/components/OpportunityModal/OpportunityModal.css

```diff
@@ -0,0 +1,31 @@
+.oppmodal {
+    align-items: center;
+    background-color: rgba(0, 0, 0, 0.4);
+    color: #f5f5f5;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    left: 0;
+    overflow: auto;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 1000;
```

Comment:
Probably not a huge deal for the scope of this project, but this often ends up in an `z-index` arms race. It'd be good to either move the z-index value to a global css file and add `.modal` class to this modal. 

Also, if you plan on having multiple modals, could be helpful to have a generic `Modal` component, which can handle the show/hide behavior, and you could pass in custom content and any generic callbacks like `onClose`.